mobile-app/utils.ts

```diff
@@ -0,0 +1,5 @@
+export const DEFAULT_CELL_COLOR = "black";
+export const PLAYER_1_CELL_COLOR = "blue";
+export const PLAYER_2_CELL_COLOR = "red";
+
+export type CellType = "player1Border" | "player2Border" | "playable";

```

Comment:
```suggestion
export type RenderedCellType = "player1Border" | "player2Border" | "playable";

```

This is a suggestion (I also could not find a better name), because I fear this could be confusing with our already existing `Cell` interface.